test/utils/image/manifest.go

```diff
@@ -19,7 +19,9 @@ package image
 import (
 	"fmt"
 	"io/ioutil"
+	"log"
```

Comment:
Please use k8s.io/klog as in the rest of Kubernetes (if there is logging instead of error reporting, see other comment).